homeassistant/components/sonos/alarms.py

```diff
@@ -27,14 +26,14 @@ class SonosAlarms(SonosHouseholdCoordinator):
     def __init__(self, *args: Any) -> None:
         """Initialize the data."""
         super().__init__(*args)
-        self.alarms: Alarms = Alarms()
+        self.alarms: alarms.Alarms = alarms.Alarms()
```

Comment:
Casually looking over the code -> I prefer the previous version better. Is there a reason you dont want to import 

Alarms and Alarm? 

Does it impact the typing somehow?